backend/apps/ai/management/commands/ai_create_event_chunks.py

```
@@ -0,0 +1,168 @@
+        else:
+            queryset = Event.upcoming_events()
+
+        if not (total_events := queryset.count()):
+            self.stdout.write("No events found to process")
+            return
+
+        self.stdout.write(f"Found {total_events} events to process")
+
+        batch_size = options["batch_size"]
+        for offset in range(0, total_events, batch_size):
+            batch_events = queryset[offset : offset + batch_size]
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)